soundcraft/gui.py

```diff
@@ -237,9 +257,41 @@ def close_cb(self, action, parameter):
 
 class App(Gtk.Application):
     def __init__(self):
-        super().__init__(application_id="soundcraft.utils")
+        super().__init__(
+            application_id=const.APPLICATION_ID,
+            flags=(
+                0
+                | Gio.ApplicationFlags.HANDLES_COMMAND_LINE  # noqa: W503
+                | Gio.ApplicationFlags.NON_UNIQUE  # noqa: W503
```

Comment:
I don't feel to strongly about it either way... My understanding of this flag is that it would prevent more than one gui from running per session bus; which would effectively mean a 1:1 gui-to-service mapping.  That seems reasonable to do, but not really necessary either.

Do you have a good use-case in mind for why we would want `NON_UNIQUE` set?

Reply:
I can think of two use cases for which I might want to use more than one GUI window:

  * Use two GUI windows side by side to make sure both the GUI and the D-Bus service work as intended, when I click on "Apply" in one window, and the other window updates instantly. Mainly interesting for developers.

  * Use one GUI window on one virtual desktop, and another GUI window on another virtual desktop. Depending on how I have arranged my applications on my set of virtual desktops, this might be useful. Another widely used tool for audio settings, namely `pavucontrol` allows me to do just that. Interesting for users.

As `soundcraft_gui` is significantly less resource intensive than something like, say, libreoffice, having multiple instances running at the same time does not appear to be a resource issue either.